service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderControllerMidEventTest.java

```diff
@@ -314,5 +364,15 @@ private DocumentReference expectedDraftDocument() {
                 .url(document.links.self.href)
                 .build();
         }
+
+        private Element<Child> childWithFinalOrderIssued(String finalOrderIssued) {
```

Comment:
Can maybe put this in TestDataHelper (next to testChild()) as we use it in both controller + service tests and it may be used more in the future

Reply:
Are we remaking CaseDataGeneratorHelper 😆

Reply:
Least it's no longer in main ;)

Reply:
New and improved! (I think for something simple like this it is ideal, but yes let's try not to recreate that!)